sandbox/api/app.py

```diff
@@ -34,35 +61,131 @@ def get_related_persons() -> Union[dict, tuple]:
         if errors := check_for_errors(request):
             return errors
 
+        identifier = request.args.get("identifier")
+        patient_identifier = request.args.get("patient:identifier")
+        include = request.args.get("_include")
+
+        if empty := __check_for_empty(identifier, patient_identifier):
+            return empty
+
         # Successful request, select response
-        if (
-            request.args.get("identifier")
-            and request.args.get("patient")
-            and request.args.get("_include") == "RelatedPerson:patient"
+        if zero_nine := __check_for_validate(
+            "9000000009",
+            identifier,
+            patient_identifier,
+            include,
+            VALIDATE_RELATIONSHIP_009,
+            VALIDATE_RELATIONSHIP_INCLUDE_009,
         ):
-            # Request with identifier, patient and _include=patient
-            return get_response(
-                "./api/responses/GET_RelatedPerson/identifier_and_patient_identifier_include.json"
-            )
-        elif request.args.get("identifier") and request.args.get("patient"):
-            # Request with identifier and patient
-            return get_response(
-                "./api/responses/GET_RelatedPerson/identifier_and_patient_identifier.json"
-            )
-        elif (
-            request.args.get("identifier")
-            and request.args.get("_include") == "RelatedPerson:patient"
+            return zero_nine
+
+        if two_five := __check_for_validate(
+            "9000000025",
+            identifier,
+            patient_identifier,
+            include,
+            VALIDATE_RELATIONSHIP_025,
+            VALIDATE_RELATIONSHIP_INCLUDE_025,
+        ):
+            return two_five
+
+        if one_seven := __check_for_list(
+            "9000000017",
+            identifier,
+            include,
+            LIST_RELATIONSHIP,
+            LIST_RELATIONSHIP_INCLUDE,
         ):
-            # Request with identifier and _include=patient
-            return get_response(
-                "./api/responses/GET_RelatedPerson/identifier_include.json"
-            )
-        elif request.args.get("identifier"):
-            # Request with identifier
-            return get_response("./api/responses/GET_RelatedPerson/identifier.json")
-        else:
-            raise ValueError("Invalid request")
+            return one_seven
+
+        raise ValueError("Invalid request")
 
     except Exception as e:
         logger.error(e)
-        return get_response("./api/responses/internal_server_error.json"), 500
+        return __generate_response(load_json_file(ERROR_RESPONSE), 500)
+
+
+def __check_for_empty(identifier: str, patient_identifier: str) -> Response:
```

Comment:
Might be worth migrating all new functions to another file so only the app routes remain in this app.py. Thinking about when the sandbox will have multiple routes

Reply:
I will migrate these to a separate file or utils.